src/common/options/osd.yaml.in

```diff
@@ -520,6 +520,64 @@ options:
     stats (inc. scrub/block duration) every this many seconds.
   default: 120
   with_legacy: false
+- name: osd_scrub_retry_delay
+  type: int
+  level: advanced
+  desc: Period (in seconds) before retrying a specific PG following a scrub failure
+  long_desc: Minimum delay after a failed attempt to scrub a PG. See the
+    'see also' for the configuration options for some specific delay reasons
```

Comment:
nit: suggest
>     'see also' for delay options for specific failure reasons.

I also wonder if `osd_scrub_retry_delay` overrides the below, if it applies to only cases that aren't among the below, or what.  In other words, I'd like to see more about the relationship between this first option and the below.